src/hooks/player_management/assign_physical_index.cpp

```diff
@@ -128,9 +128,18 @@ namespace big
 
 					if (g.session.lock_session && g_player_service->get_self()->is_host())
 					{
-						dynamic_cast<player_command*>(command::get(RAGE_JOAAT("breakup")))->call(plyr, {});
-						g_notification_service->push_warning("Lock Session",
-						    std::format("A player with the name of {} has been denied entry", plyr->get_net_data()->m_name));
+						if (plyr.is_friend() && g.session.allow_friends_into_locked_session)
+						{
+							g_notification_service->push_success("Lock Session",
+							    std::format("A player friended player with the name of {} has been allowed to join the locked lobby!",
```

Comment:
We call it a session and don't use `!` anywhere else so we probably shouldn't here.